charts/seed-operatingsystemconfig/original/templates/osc.yaml

```diff
@@ -32,3 +42,9 @@ spec:
 {{ include "kernel-config" . | indent 2 }}
 {{ include "health-monitor" . | indent 2 }}
 {{ include "gardener-user-script" . | indent 2 }}
+{{- if .Values.enableContainerD }}
+{{ include "init-containerd-config-script" . | indent 2 }}
+{{- end }}
+
```

Comment:
Can we remove the unnecessary newlines at the end here? :)

Reply:
Sure